config/config_mgmt.py

```diff
@@ -555,27 +561,30 @@ def testRun_Delete_Add_Ports():
         return None
 
     print("\n***Port Deletion Test Passed***\n")
+    import time
+    time.sleep(5)
 
     portJson = {
         "PORT": {
             "Ethernet0": {
                     "alias": "Eth1/1",
                     "admin_status": "up",
-                    "lanes": "65, 66",
+                    "lanes": "65,66",
                     "description": "",
                     "speed": "50000"
             },
             "Ethernet2": {
                     "alias": "Eth1/3",
                     "admin_status": "up",
-                    "lanes": "67, 68",
+                    "lanes": "67,68",
                     "description": "",
                     "speed": "50000"
             }
         }
     }
 
-    ret = cm.addPorts(ports=['Ethernet0', 'Ethernet2'], portJson=portJson)
+    #ret = cm.addPorts(ports=['Ethernet0', 'Ethernet2'], portJson=portJson, loadDefConfig=False)
+    ret = cm.addPorts(ports=['Ethernet0', 'Ethernet2'], portJson=portJson, loadDefConfig=True)
```

Comment:
loadDefConfig = False case  will be tested too?

Reply:
Changed the Test Code.